src/rpc/blockchain.cpp

```diff
@@ -2746,7 +2746,15 @@ static RPCHelpMan loadtxoutset()
     }
 
     SnapshotMetadata metadata;
-    afile >> metadata;
+    try {
+        // Read data from the file
+        afile >> metadata;
+    } catch (const std::exception& e) {
```

Comment:
To fix the MSVC CI job, suggesting:

```suggestion
    } catch (const std::exception&) {
```

Reply:
done, thanks!